src/colvarbias_abf.h

```diff
@@ -184,6 +196,38 @@ class colvarbias_abf : public colvarbias {
 
   /// Calculate the bias energy for 1D ABF
   virtual int calc_energy(std::vector<colvarvalue> const *values);
-};
 
+  /// Initialize specific dependencies of ABF derived class
+  /// Adding them to those of base class colvarbias
+  /// Alternately we could overload the init_dependencies() function
+  virtual int init_dependencies();
+
+  enum features_bias_abf {
+    /// Start after generic cvb features
+    /// There is at least one ext-Lagrangian colvar -> run eABF
+    f_cvb_abf_extended = f_cvb_ntot,
+    /// Total number of features for an ABF bias
+    f_cvb_abf_ntot
+  };
```

Comment:
This change looks more or less okay to me with the code as it is. However, because we can finally rely on C++11 or later, I think it would be great to use `enum class` for the dependency features, and this kind of initialization would prevent it.

On the other hand, we could stick with a simple `int` based `enum`, but if another bias had its own set of specialized features, its values would clash with the ABF ones.

Lastly, the extended Lagrangian was even introduced originally for metadynamics (before most people realize it doesn't add much benefit). So it seems be more perplexing to me to introduce sub-features just for this particular example.

How about adding instead `f_cvb_extended_Lagriangian` as a generic bias feature?

Reply:
I second the use of `enum class`.

Reply:
Initially I did this extension to add experimental features that have been removes since. I am not worried about clashes, because the extended features of each derived class would only be used within the derived class, so reusing the same values would have no consequence. But it would interfere with moving to enum class. This one feature can certainly be added to the generic set. I suppose that even features that are unique to a single derived class can be added to the generic set, and unused by most derived classes. It seems like pollution of the base class to me, though.

Reply:
Change done.